spec/views/two_factor_authentication/recovery_code_verification/show.html.slim_spec.rb

```diff
@@ -31,4 +31,9 @@
     expect(rendered).
       to have_xpath("//form[@method='post']")
   end
+
+  it 'allows the user to cancel out of the sign in process' do
+    render
+    expect(rendered).to have_link(t('links.cancel'), destroy_user_session_path)
```

Comment:
maybe not worth it yet, but it might be nice to put these tests, since they are all exactly the same, into a shared example group: https://www.relishapp.com/rspec/rspec-core/docs/example-groups/shared-examples